app/components/search/SearchResults/SearchResults.tsx

```diff
@@ -64,6 +64,13 @@ const SearchResults = ({
     );
   };
 
+  function handleAction(searchMapAction: SearchMapActions) {
```

Comment:
**question (non-blocking):** any reason to not use an arrow function for consistency?

Reply:
I'll change it. I think I just didn't notice what I was doing.